tool/getInformationReadyForRegisterBtcTransaction.js

```diff
@@ -0,0 +1,49 @@
+const mempoolJS = require("@mempool/mempool.js");
+const pmtBuilder = require("../index");
+
+const getInformationReadyForRegisterBtcTransaction = async (transactionHash, network = 'testnet') => {
```

Comment:
The network will probably not be optional, since it's received before the tx hash. So it should always be provided